RecoParticleFlow/PFProducer/interface/PFEGammaFilters.h

```diff
@@ -56,24 +57,32 @@ class PFEGammaFilters {
   const unsigned int ele_missinghits_;
   const float ele_ecalDrivenHademPreselCut_;
   const float ele_maxElePtForOnlyMVAPresel_;
-  const float ele_maxNtracks, ele_maxHcalE, ele_maxTrackPOverEele, ele_maxE,
-    ele_maxEleHcalEOverEcalE, ele_maxEcalEOverPRes, ele_maxEeleOverPoutRes,
-    ele_maxHcalEOverP, ele_maxHcalEOverEcalE, ele_maxEcalEOverP_1,
-    ele_maxEcalEOverP_2, ele_maxEeleOverPout, ele_maxDPhiIN;
+  float ele_maxNtracks_;
+  float ele_maxHcalE_;
+  float ele_maxTrackPOverEele_;
+  float ele_maxE_;
+  float ele_maxEleHcalEOverEcalE_;
+  float ele_maxEcalEOverPRes_;
+  float ele_maxEeleOverPout_Res_;
+  float ele_maxHcalE_OverP_;
+  float ele_maxHcalE_OverEcalE_;
+  float ele_maxEcalEOverP_1_;
```

Comment:
what was the logic behind inserting underscores in the middle of the data member names.
I'm missing the pattern: sometimes ```Res``` is separated, sometimes ```Over``` is separated. If the underscores in the middle are really needed, the capitalization is not clear.

May I suggest to stay with simply adding an underscore.
(even reverting back to no trailing underscore would probably be better than this new naming pattern which differs in a non-trivial way from the config parameter names).

Reply:
I have no idea which version of `%s/foo/bar/g` in vim was leading to that atrocity... of course I only meant to add single underscores.